mautic/mautic
# Fix display zero values in reports

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 4.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
Value 0 of boolean or select  contact fields are not displayed in reports. This fixed it



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create custom select and boolean field
3. Custom select create with these values
![image](https://user-images.githubusercontent.com/462477/122730724-d5d0b500-d27a-11eb-8398-71068a9a93b5.png)
4. Then create contact with 
![image](https://user-images.githubusercontent.com/462477/122730902-0c0e3480-d27b-11eb-983a-de63647fbe5d.png)
5. Then create report for this contact and you see
![image](https://user-images.githubusercontent.com/462477/122731056-32cc6b00-d27b-11eb-91e5-7261cb94027e.png)
6. After PR you see
![image](https://user-images.githubusercontent.com/462477/122731198-542d5700-d27b-11eb-8088-bc13d185fdd3.png)
